data/sql/updates/pending_db_world/rev_1588410722246016400.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1588410722246016400');
+
+UPDATE `creature_template` SET `lootid` = '0' WHERE (`entry` = '2992', '7484', '3680', '17079', '17080');
```

Comment:
`lootid` is a number, not a string, same for `entry` values

Reply:
+ the entry change makes no sense.
go back to WHERE `entry` IN (nnnn)